falkordb-node/node-entrypoint.sh

```
@@ -34,10 +34,22 @@ PERSISTENCE_AOF_CONFIG=${PERSISTENCE_AOF_CONFIG:-'everysec'}
 FALKORDB_CACHE_SIZE=${FALKORDB_CACHE_SIZE:-25}
 FALKORDB_NODE_CREATION_BUFFER=${FALKORDB_NODE_CREATION_BUFFER:-16384}
 FALKORDB_MAX_QUEUED_QUERIES=${FALKORDB_MAX_QUEUED_QUERIES:-50}
-FALKORDB_TIMEOUT_MAX=${FALKORDB_TIMEOUT_MAX:-0}
-FALKORDB_TIMEOUT_DEFAULT=${FALKORDB_TIMEOUT_DEFAULT:-0}
 FALKORDB_RESULT_SET_SIZE=${FALKORDB_RESULT_SET_SIZE:-10000}
 FALKORDB_QUERY_MEM_CAPACITY=${FALKORDB_QUERY_MEM_CAPACITY:-0}
+FALKORDB_TIMEOUT_MAX=${FALKORDB_TIMEOUT_MAX:-0}
+FALKORDB_TIMEOUT_DEFAULT=${FALKORDB_TIMEOUT_DEFAULT:-0}
+
+# If vars are <nil>, set it to 0
+if [[ "$FALKORDB_QUERY_MEM_CAPACITY" -eq "<nil>" ]]; then
```

Shellcheck

[warning] -eq treats this as an arithmetic expression. Use = to compare as string (or expand explicitly with $((expr))).

(SC2309)